string-template/src/test/java/org/jdbi/v3/sqlobject/BindInTest.java

```diff
@@ -65,6 +65,25 @@ public static void exit()
     //
 
     @Test
+    public void testSomethingWithExplicitAttributeName()
+    {
+        final SomethingWithExplicitAttributeName s = handle.attach(SomethingWithExplicitAttributeName.class);
+
+        final List<Something> out = s.get(1, 2);
+
+        assertThat(out).hasSize(2);
+    }
+
+    @UseStringTemplateStatementRewriter
+    public interface SomethingWithExplicitAttributeName
+    {
+        @SqlQuery("select id, name from something where id in (<ids>)")
+        List<Something> get(@BindIn("ids") int... ids);
```

Comment:
If the point here is to test the explicit setting of `ids`, it should probably differ from the parameter name?